.stylelintignore

```diff
@@ -2,3 +2,5 @@
 **/_site/**/*.css
 packages/cli/test/functional/**/expected/**/*.css
 packages/vue-components/**/*.css
+packages/cli/coverage/**/*.css
+packages/core/coverage/**/*.css
```

Comment:
does this mean that when tests are running locally the coverage artifacts will be generated? do we need to gitignore them as well?

Reply:
Hi @tlylt, the current `.gitignore` already ignores the coverage files.